modules/widgets/wordpress-post-widget.php

```diff
@@ -79,7 +79,7 @@ function jetpack_display_posts_update_cron_action() {
  * @param string $plugin_file_name The plugin file that was activated.
  */
 function jetpack_conditionally_activate_cron_on_plugin_activation( $plugin_file_name ) {
-	if ( 'jetpack/jetpack.php' === $plugin_file_name ) {
+	if ( plugin_basename(JETPACK__PLUGIN_FILE) === $plugin_file_name ) {
```

Comment:
It looks to me now that the entire method doesn't work because the `activated_plugin` hook does not run if it's set in the plugin that is being currently activated, much like the `register_activation_hook` method:
https://codex.wordpress.org/Function_Reference/register_activation_hook

It seems to me that we need some another way of activating cron on Jetpack activation.